rllib/BUILD

```diff
@@ -3311,31 +3311,31 @@ py_test(
 py_test(
     name = "examples/bandit/lin_ts_train_wheel_env",
     main = "examples/bandit/lin_ts_train_wheel_env.py",
-    tags = ["team:rllib", "exclusive", "examples"],
+    tags = ["team:rllib", "exclusive", "examples", "examples_B"],
```

Comment:
Is examples_B already a valid tag? Why are you adding this tag?

Reply:
yeah it is. we have to add these examples_X tags man, otherwise, they don't get included in any of the runs.
I discovered it this weekend too ...